build.xml

```diff
@@ -402,38 +402,42 @@
     </if>
 
     <!-- build and configure the requested database type -->
+    <phingcall target="setup_database" />
+
+    <!-- start Solr (use restart in case of old PID files) -->
```

Comment:
This comment should probably move inside the setup_solr task since it refers to an implementation detail.

Reply:
Moved.